snippets/microshift-update-paths-snip.adoc

```diff
@@ -6,24 +6,24 @@
 :_mod-docs-content-type: SNIPPET
 
 [id="microshift-about-updates-checking-version-update-path_{context}"]
-= Checking version update path
+= Checking the {microshift-short} version update path
 
 Before updating {microshift-short} or {op-system}, determine the compatibilities. Plan for the versions of each that you intend to use.
 
 *{product-title} update paths*
 
 {microshift-short} version 4.17::
-* Version 4.17 to 4.17.z on {op-system} or {op-system-ostree} 9.4
+* Version 4.17{nbsp}&#8594;{nbsp}4.17.z on {op-system} 9.4
 
 {microshift-short} version 4.16::
-* Version 4.16 on {op-system} or {op-system-ostree} 9.4 to 4.17 on {op-system} or {op-system-ostree} 9.4
-* Version 4.16 to 4.16.z on {op-system} or {op-system-ostree} 9.4
+* Version 4.16{nbsp}&#8594;{nbsp}4.17 on {op-system} 9.4
+* Version 4.16{nbsp}&#8594;{nbsp}4.16.z on {op-system} 9.4
 
 {microshift-short} version 4.15::
-* Version 4.15 on {op-system} or {op-system-ostree} 9.2 or 9.3 to 4.16 on {op-system} or {op-system-ostree} 9.4
-* Version 4.15 to 4.15.z on {op-system} or {op-system-ostree} 9.2 or 9.3
+* Version 4.15 on {op-system} 9.2 or 9.3{nbsp}&#8594;{nbsp}4.16 on {op-system} 9.4
+* Version 4.15{nbsp}&#8594;{nbsp}4.15.z on {op-system} 9.2 or 9.3
 
 {microshift-short} version 4.14::
-* Version 4.14 on {op-system} or {op-system-ostree} 9.2 or 9.3 to 4.16 on {op-system} or {op-system-ostree} 9.4
-* Version 4.14 to 4.15 on {op-system} or {op-system-ostree} 9.2 or 9.3
-* Version 4.14 to 4.14.z on {op-system} or {op-system-ostree} 9.2 or 9.3
\ No newline at end of file
+* Version 4.14 on {op-system} 9.2 or 9.3 to 4.16 on {op-system} 9.4
```

Comment:
Won't stop the merge review over this unless you want to, but it seems like you missed an instance of converting "to" to the arrow icon here.

Reply:
Ah durn. I'll fix it.